packages/stark-ui/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@angular/material-moment-adapter": "^7.0.0",
     "@mdi/angular-material": "^3.3.92",
+    "@types/lodash-es": "^4.17.1",
```

Comment:
Shouldn't the types from `lodash-es` be taken from stark-core by changing the path in the `tsconfig.json` of stark-ui ?

Reply:
I will try again, but I think my IDE didn't recognize the types when I tried that. 🤔

Reply:
I'm hitting a lot of other issues. Maybe this is more suitable for a separate ticket? 😕